nservicebus/upgrades/6to7/index.md

```diff
@@ -161,6 +161,17 @@ The [.Retries message receiver](/nservicebus/recoverability/configure-delayed-re
 
 The [MSMQ transport](/transports/msmq) is no longer part of the NServiceBus NuGet package. It has been moved into a separate package, [NServiceBus.Transport.Msmq](https://www.nuget.org/packages/NServiceBus.Transport.Msmq/).
 
+### Provision of PowerShell Scripts
+
+To facilitate the queue creation for endpoints during deployment, the NuGet package now comes packaged with a `Scripts` folder with two new scripts called, `CreateQueues.ps1` and `DeleteQueues.ps1`. These can be located as part of the package. For example, under `C:\Users\username\.nuget\packages\nservicebus.transport.msmq\1.0.0\lib\net452\Scripts`. 
```

Comment:
Should 'the' be dropped?
> To facilitate queue creation for endpoints
or
> To facilitate the creation of queues for endpoints

But I'm not native speaker and likely to be wrong :-)

Reply:
@ramonsmits - Changed it to "to facilitate the creation of queues" per your suggestion.